test/js_optimizer/emitDCEGraph6.js

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
I wonder if we can give this (and the other tests 1-5) better names?  Can be followup maybe?

Reply:
I'll rename the new one in this PR. I can look at the others later.